pyproject.toml

```diff
@@ -32,21 +32,8 @@ flake8-max-line-length = 110
 flake8-max-doc-length = 79
```

Comment:
Please remove lines 29 to 32. They are not needed anymore.

Reply:
Done in the commit: [33fd78c](https://github.com/lsst-ts/ts_m2com/pull/30/commits/33fd78cc2bff205ca6cdc1cfaea0c286f7b3f08f). Thanks!